modules/stanford_events/lib/scss/components/event-list/stanford_events.event-list.scss

```diff
@@ -27,6 +27,16 @@ h1 {
   @include modular-spacing('margin', null null 5 null);
 }
 
+.stanford-events-list-page {
+  &.list-page {
+    .su-list-unstyled > li:last-child {
+      article {
```

Comment:
Do we need this much specificity? this is really alot.

Reply:
I'll look. I laughed because right after I read this, you said in another, is this specific enough?  LOL

Reply:
@pookmish Ok, I made it lot less specific and it is still working as I want.